### PR TITLE
feat: styled and shaped the whole thing Teej

### PR DESCRIPTION
**Pull request by wildduck2**
image

i hope you like it just check the css file to make sure you customize the colors and all the sudo els

### CODE REVIEW

File: mixery/assets/css/app.css [hunk 1]
```diff
@@ -3,3 +3,177 @@
 @import "tailwindcss/utilities";
 
 /* This file is for your main application CSS */
+
+/*
+* INFO: this is the style from wild_duck(ahmed_ayob) 
+* */
+
+@import url("https://fonts.googleapis.com/css2?family=Lexend+Deca:wght@100..900&family=Noto+Sans+Arabic:wght@100..900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
```
Comment (jonnytest1) [inline mixery/assets/css/app.css:11]: this isnt conform with eu data protection laws :3 (gotta proxy though server so ip isnt leaked)

Comment (wildduck2) [reply mixery/assets/css/app.css:11]: does google does that ?? 
i have heard this before so i should notn't use google fonts ??

Comment (jonnytest1) [reply mixery/assets/css/app.css:11]: you can use it , but not directly - you can probably just proxy the request through our own backend , or downlaod it and include the font in your own assets
(its not really about if google stored the ip , but that the possiblity exists for them to theoretically do that ^^ )

**Comment by jonnytest1** [comment]
did you already collect your teej coin for making a PR ? :3

File: mixery/assets/css/app.css (hunk 1 above)
Comment (wildduck2) [reply mixery/assets/css/app.css:11]: that is a good one

**Comment by wildduck2** [comment]
> did you already collect your teej coin for making a PR ? :3

i do not really care about coins i am  just helping teej cause he is a good person period blank :sligh_smile:

**Comment by jonnytest1** [comment]
> > did you already collect your teej coin for making a PR ? :3
> 
> i do not really care about coins i am just helping teej cause he is a good person period blank :sligh_smile:

just for the memes tho :D

**Pull request merged**